ocaml/libs/xapi-rrd/lib/rrd.ml

```diff
@@ -22,6 +22,12 @@ exception No_RRA_Available
 
 exception Invalid_data_source of string
 
+(** Inverse is (fun x -> 1.0 - x) *)
+type ds_transform_function = Inverse | Identity
```

Comment:
How about 
```
type transform = Inv of float -> float | Id of float -> float
```
that is, make the actual transformation part of the type?

Reply:
I'd be wary of adding more information in the pages, we're filling the memory pages, for not a big benefit. I'm even thinking whether serializing it as a number to reduce the amount of bytes added would be worth it (0 being Identity, 1 the inverse)

Reply:
then it'd need to be serialized. the intention is to make this as simple as possible to transfer between plugin and server